### ISSUE TITLE
Allow symfony/yaml:^6

### ISSUE THREAD

**Issue by KorDum** [VKCOM/modulite-phpstan#18]
Hello!
Can you allow Symfony Yaml package version as `^5.4 || ^6` please?

```
  Problem 1
    - Root composer.json requires vkcom/modulite-phpstan * -> satisfiable by vkcom/modulite-phpstan[v1.0.0].
    - vkcom/modulite-phpstan v1.0.0 requires symfony/yaml ^5.4 -> found symfony/yaml[v5.4.0, ..., v5.4.17] but the package is fixed to v6.2.2 (lock file version) by a partial update and that version does not match. Make sure you list it as an argument for the update command.
```

**Comment by KorDum** [comment]
There is pull request
https://github.com/VKCOM/modulite-phpstan/pull/12

**Comment by Danil42Russia** [comment]
@KorDum, hi!
We fixed the `symfony/yaml` version problem in #42. Do we close your issue?
